example/widgetbook/pages/components/snack_bar_widgetbook.dart

```diff
@@ -0,0 +1,59 @@
+import 'package:flutter/material.dart';
+import 'package:widgetbook/widgetbook.dart';
+import 'package:zeta_flutter/zeta_flutter.dart';
+
+import '../../test/test_components.dart';
+
+Widget snackBarUseCase(BuildContext context) {
+  return WidgetbookTestWidget(
+    widget: Builder(
+      builder: (context) {
+        final text = context.knobs.string(
+          label: 'Content Text',
+          initialValue: 'This is a snackbar',
+        );
+
+        final actionLabel = context.knobs.stringOrNull(
+          label: 'Action Label',
+          initialValue: null,
+        );
+
+        final type = context.knobs.listOrNull<ZetaSnackBarType>(
+          label: 'Type',
+          options: [null, ...ZetaSnackBarType.values],
+          labelBuilder: (type) => type?.name ?? '',
+        );
+
+        final leadingIcon = context.knobs.listOrNull<Icon>(
```

Comment:
There is a helper function `iconKnob` that can add any custom icon, that might be good to use here